genesyscloud/data_source_genesyscloud_outbound_settings.go

```diff
@@ -0,0 +1,51 @@
+package genesyscloud
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mypurecloud/platform-client-sdk-go/v80/platformclientv2"
+)
+
+func dataSourceOutboundSettings() *schema.Resource {
+	return &schema.Resource{
+		Description:   "An organization's outbound settings",
+		ReadContext:   readWithPooledClient(dataSourceOutboundSettingsRead),
+		SchemaVersion: 1,
+		Schema:        resourceOutboundSettings().Schema,
```

Comment:
I'm not sure if there is any need for this data source if the resource is for defining default behaviours. @carnellj-genesys thoughts? 
Also I'm getting a panic error when I try to create this data source with no fields set, similar to how we do with the data source `genesyscloud_auth_division_home`

Reply:
When I was creating the routing settings, which I think is very similar to the outbound settings, Ronan thought I should create a data source but I can remove it if we don't think it's necessary

Reply:
I am ok with leaving the data source out.